example/client/http_client.cpp

```
@@ -34,13 +36,14 @@ do_request(
             stream, capy::const_buffer(request.data(), request.size())); ec)
         throw boost::system::system_error(ec);
 
-    // Read the entire response
+    // Read the entire response until EOF
     std::string response;
-    auto [ec, n] = co_await corosio::read(stream, response);
+    auto [ec, n] = co_await capy::read(
+            stream, capy::string_dynamic_buffer(&response));
     // EOF is expected when server closes connection
     if (ec && ec != capy::error::eof)
         throw boost::system::system_error(ec);
```

Cppcheck (2.19.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)